src/com/cdd/bao/editor/DetailPane.java

```diff
@@ -75,7 +68,8 @@
 	{
 		Lineup line;
 		Schema.Value sourceVal;
-		TextField fieldURI, fieldName;
+		URIRowLine fieldURI;
```

Comment:
Multiple declarations on the same line are allowed (though not required), so roll back the change.

Reply:
Type has changed

Reply:
Right.